core/src/commonMain/kotlin/com/ustadmobile/core/view/SelectFileView.kt

```diff
@@ -12,11 +12,20 @@ interface SelectFileView : UstadView {
 
         const val SELECTION_MODE_GALLERY = "video/*;audio/*"
 
+        const val SELECTION_MODE_VIDEO = "video/*"
```

Comment:
It would be better to have a single argument that provides the actual list of mime types. In the case of selecting a file for a content entry, this should come from the contentpluginmanager to get a list of all supported types (instead of duplicating this in a constant).